src/test/java/org/broadinstitute/hellbender/engine/dataflow/transforms/RefBasesFromAPIUnitTest.java

```diff
@@ -43,11 +39,13 @@
 
 public final class RefBasesFromAPIUnitTest extends BaseTest {
 
-    private RefAPISource createMockRefAPISource( final List<SimpleInterval> intervals ) {
-        RefAPISource mockSource = mock(RefAPISource.class, withSettings().serializable());
+    private ReferenceDataflowSource createMockReferenceDataflowSource(final List<SimpleInterval> intervals,
```

Comment:
Let's leave this to a later refactoring if that's OK with you - this change is already pretty large.